clients/pkg/logentry/stages/match.go

```diff
@@ -114,11 +114,17 @@ func newMatcherStage(logger log.Logger, jobName *string, config interface{}, reg
 }
 
 func getDropCountMetric(registerer prometheus.Registerer) *prometheus.CounterVec {
+	return getLogEntryMetric(registerer, "dropped_lines_total",
+		"A count of all log lines dropped as a result of a pipeline stage",
+		[]string{"reason"})
+}
+
+func getLogEntryMetric(registerer prometheus.Registerer, name, help string, labels []string) *prometheus.CounterVec {
 	dropCount := prometheus.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "logentry",
-		Name:      "dropped_lines_total",
```

Comment:
It would be better if this PR can not modify `match.go`. Other parts LGTM

Reply:
removed any changes from match.go